dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
+        """Returns the most recent package revision, for a
+        given package version if specified.
+
+        :param package_version: a given package version
+        :type package_version: str
+        :returns: package revision
+        :rtype: int
+        """
+
+        if package_version:
+            software_versions = self.software_versions()
+            revision = next(
+                (pkg_ver for pkg_ver
+                    in reversed(software_versions)
```

Comment:
Maybe add a note that this depends on `software_versions()` returning an `OrderedDict`.